docs/edpm.md

```diff
@@ -0,0 +1,34 @@
+# EDPM
+
+Deploying EDPM with network isolation
+
+### Dev setup
+Setting the node(s)
+```shell
+cd devsetup
+make crc
+make crc_attach_default_interface
+EDPM_COMPUTE_SUFFIX=0 make edpm_compute
+EDPM_COMPUTE_SUFFIX=0 make edpm_compute_repos
+EDPM_COMPUTE_SUFFIX=1 make edpm_compute
+EDPM_COMPUTE_SUFFIX=1 make edpm_compute_repos
+
+# check that the two computes got and IP suffix of 100 and 101 respectively as the later steps depends on that
+
+cd ..
+```
+### Deploy
+```shell
+make crc_storage
+make ceph TIMEOUT=90 # (optional)
```

Comment:
When we deploy ceph, if we're using network isolation it should be deployed this way.

`TIMEOUT=90 HOSTNETWORK=false NETWORKS_ANNOTATION=\'[\{\"name\":\"storage\",\"namespace\":\"openstack\"\}]\' MON_IP=172.18.0.30 make ceph`